src/test/java/org/json/junit/XMLTest.java

```diff
@@ -1247,6 +1249,39 @@ public void testIndentComplicatedJsonObjectWithArrayAndWithConfig(){
+
+        Throwable throwable = assertThrows(JSONException.class, new ThrowingRunnable() {
+
+            @Override
+            public void run() throws Throwable {
+                XML.toJSONObject(wayTooLongMalformedXML, XMLParserConfiguration.ORIGINAL.withMaxNestingDepth(42));
+            }
+        });
+
+        assertTrue("Wrong throwable thrown: not expecting message <" + throwable.getMessage() + ">", throwable.getMessage().startsWith("Maximum nesting depth of"));
+
+        final String perfectlyFineXML = "<Test>\n" +
```

Comment:
In this class, the standard is to have one test case (test xml) per test method. It would be more consistent to adhere to the established convention, and create multiple test methods.